examples/tdec/simple_tdec_local.py

```diff
@@ -0,0 +1,28 @@
+from examples.tdec.demo_utilities.demo_sample_conditions import ten_oclock_florida_time
+from nucypher.characters.chaotic import BobGonnaBob, NiceGuyEddie
+
+plaintext = b"PEACE AD DAWN"
+THIS_IS_NOT_A_TRINKET = 55
+
+enrico = NiceGuyEddie(encrypting_key=THIS_IS_NOT_A_TRINKET)
+bob = BobGonnaBob(domain="lynx")
```

Comment:
You almost have a crony theme here, but bob needs a better name (`Bobbykins`?).  Or for the unimaginative, `LocalBob` and `LocalEnrico`

Reply:
Robert

Reply:
https://getyarn.io/yarn-clip/c8fe0950-378c-4fc4-a417-b08b61441ec6